config/application.rb

```diff
@@ -21,6 +21,7 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     config.eager_load_paths << Rails.root.join("app/workers/concerns")
+    config.eager_load_paths << Rails.root.join("lib/github_service/concerns")
```

Comment:
I thikn we can avoid this concerns directory by just putting it in `github_service/is_team_member.rb` directly, then just doing an `include GithubService::IsTeamMember`

Reply:
We can, I was just trying to follow the `concerns/` convention.  I have hard opinions on where it lives.